File: programers_Lv2/최댓값과 최솟값.js
function solution(s) {
  var answer = "";

  const arr = s.split(" ").map((el) => parseInt(el));

  let minVal = arr.sort((a, b) => a - b)[0];
  let maxVal = arr.sort((a, b) => b - a)[0];

  answer = `${minVal} ${maxVal}`;

  return answer;
}

console.log(solution("1 2 3 4"));

File: programers_Lv2/최댓값과 최솟값.py
def solution(s):
    answer = ''

    lst = list(map(int, s.split()))

    min_rst = min(lst)
    max_rst = max(lst)

    answer = f'{min_rst} {max_rst}'

    return answer

print(solution("1 2 3 4"))
print(solution("-1 -2 -3 -4"))